csunplugged/topics/content/en/kidbots/programming-challenges/flip-coins/python-hints.md

```diff
@@ -0,0 +1,7 @@
+-   To simulate a coin toss, use the random package:
```

Comment:
Might be good to enclosed 'random' in backticks here as it's a special Python keyword.